src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
```

Comment:
On my opinion it's better to call it searchQuery or searchInput. There could be a lot of different queries in large apps, so it's always nice to use more descriptive variable names, to avoid misunderstanding.